plugins/kn-admin/cmd/private-registry/private-registry.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package private_registry
+
+import (
+	"github.com/knative/client-contrib/plugins/kn-admin/cmd"
+	"github.com/spf13/cobra"
+)
+
+// privateRegistryCmd represents the privateRegistry command
+func NewPrivateRegistryCmd(p *cmd.AdminParams) *cobra.Command {
+	var privateRegistryCmd = &cobra.Command{
+		Use:   "private-registry",
+		Short: "Manage private-registry",
+		Long: `A longer description that spans multiple lines and likely contains examples
+and usage of using your command. For example:
+
+Cobra is a CLI library for Go that empowers applications.
```

Comment:
Same as above. These three lines are not needed.

Reply:
Fixed!